dask/array/fft.py

```diff
@@ -227,3 +228,38 @@ def fftfreq(n, d=1.0, chunks=None):
     r /= d
 
     return r
+
+
+@wraps(np.fft.fftshift)
```

Comment:
This wrapper is not strictly needed here. It is only here as it is a holdover from the code being refactored. Added PR ( https://github.com/dask/dask/pull/2323 ) to address this.